### ISSUE TITLE
cheese curds not craftable

### ISSUE THREAD

**Issue by MicrodenNL** [FTBTeam/FTB-Modpack-Issues#4121]
### Modpack

FTB Presents Direwolf20 1.20

### Modpack version

1.3.1

### Describe the bug

we can not make CHeese curds in the mixing vat.
we follow  the recipe out of the growthcraft book ( JEI has no recipes for cheese curds)

### Steps to reproduce

put ingredients in mixing vat 
macking the mixing vat with a stick of sword noting work

![2023-10-26_07 17 47](https://github.com/FTBTeam/FTB-Modpack-Issues/assets/147954609/aa9cb06f-e987-458c-8c7e-b811ff881c77)

### Expected behaviour

making Cheese Curds

**Comment by natebc** [comment]
i figured out why in https://github.com/FTBTeam/FTB-Modpack-Issues/issues/4049.

TLDR; is the unification done in kubejs breaks these. if you remove 
```
    ['forge:dusts/salt', 'growthcraft:salt'],
```
From `kubejs/server_scripts/tags.js` and run `/reload`  (or restart the server). You'll be able to make cheese!

**Comment by Stathol** [comment]
> [...] if you remove
> 
> ```
>     ['forge:dusts/salt', 'growthcraft:salt'],
> ```
> 
> From `kubejs/server_scripts/tags.js` and run `/reload` (or restart the server). You'll be able to make cheese!

This will work as far as the Growthcraft mixing vat is concerned, but it also means that you're de-unifying growthcraft salt from all the other recipes in the game that call for salt. A better temporary workaround would be to exempt the mixing vat recipes from Almost Unified.

In `<instance dir>\config\almostunified\unify.json`:

```
"ignoredRecipeTypes": [
  "cucumber:shaped_tag",
  "growthcraft_milk:mixing_vat_item_recipe"
],
```

With that said...

Growthcraft rock salt ore _probably_ shouldn't exist in the first place. The pack priorities are such that Mekanism salt is the "representative" salt for unification. Nonetheless, it does exist for now, and you can use it to make cheese. Just don't craft the salt into a block, which will irreversibly turns it into Mekanism salt. Incidentally, the compacting storage drawers from Functional Storage don't really care about JEI hiding, and will happily turn Growthcraft salt dust into Growthcraft salt blocks and vice-versa.

Aside:

Railcraft Reborn gives its saltpeter the various forge salt tags, which is a bit...strange. I mean, yes, technically potassium nitrate is a "salt" in the broad chemical sense, but it's toxic and not at all the same things as table salt.

**Comment by Alatyami** [comment]
Growthcraft 9.1.2 has support for tags in Brew Kettle recipes, Mixing Vat will be compliant in a couple of days.

**Comment by Alatyami** [comment]
The fix for this on the Growthcraft side is released today in v9.1.3

**Comment by DinnerBeef** [comment]
This issue has now been fixed! Please reopen this issue if you are still having problems.
